tests/system/Database/Live/ForgeTest.php

```diff
@@ -941,7 +941,7 @@ public function testAddFields(): void
                 ],
             ];
 
-            if (version_compare($this->db->getVersion(), '8.0.17', '>=')) {
+            if (version_compare($this->db->getVersion(), '8.0.17', '>=') && str_contains($this->db->getVersion(), 'MariaDB') === false) {
```

Comment:
str_contains() not allowed for the current version of php 7.4

Reply:
I have changed `strpos` function to handle this.